CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ make build-dev
 ### Changes
 - [#348](https://github.com/LayerManager/layman/issues/348) Upgrade GeoServer to 2.21.2, because it has better rendering.
   - In workspace-specific WMS GetCapabilities documents, style name consists only of style name without `<workspace>:` prefix. For example, formerly it was `testuser_wms:blue_style`, now it is only `blues_style`.
+  - Workspace-specific WMS GetCapabilities documents includes GraphicalUrl element for every style for every layer. Previously vector layers with qml style did not have it.
```

Comment:
GraphicalUrl -> LegendURL

Reply:
qml -> QML

Reply:
I prefer to add link to https://github.com/LayerManager/layman/issues/613 .

Maybe put the line as standalone item (not subitem of GeoServer). There can be subitem saying it was caused by upgrade of GeoServer.